src/core/components/avatars/avatar/avatar.tsx

```diff
@@ -53,27 +54,31 @@ const responsiveSizeClasses: ResponsiveAttributeClassMap<AvatarSize> = {
 export const Avatar: React.FC<IAvatarProps> = (props) => {
     const { alt = 'avatar', className, fallback, responsiveSize = {}, size = 'sm', ...imageProps } = props;
 
+    const [imgLoading, setImgLoading] = useState(true);
+
     const containerClassNames = classNames(
-        'flex items-center justify-center overflow-hidden rounded-full',
+        'flex items-center justify-center overflow-hidden rounded-full [position:var(--ods-avatar-container-position)]',
```

Comment:
I think we have needed a `shrink-0` here fwiw. Almost all of our circle shapes do. Like Spinner and CircularSkeleton. They go oval in many use cases otherwise.

Reply:
Yes makes sense, I'm adding it here on the base avatar component and removing it from the components using it